src/lib/inputs/InputRadio/InputRadio.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from 'react';
+import classNames from 'classnames';
+import { FormikProps } from 'formik';
+import { FormControlLabel, FormControlLabelProps, Radio, RadioProps } from '@material-ui/core';
+
+export interface InputRadioProps extends RadioProps {
+  label?: string;
+  labelPlacement?: FormControlLabelProps['labelPlacement'];
+  formikProps?: FormikProps<any>;
+}
+
+export const InputRadio: FC<InputRadioProps> = ({
+  className,
+  checked = false,
+  color = 'primary',
+  formikProps,
+  label,
+  labelPlacement,
+  onChange,
+  value,
+  ...props
+}) => {
+  const fieldProps = formikProps?.getFieldProps(props.name);
+  const fieldValue = fieldProps?.value === value;
```

Comment:
@Carlo-Piantini you're logic for the checked prop isn't going to work I think.

we should do something like `const checked = fieldValue ?? fieldProps?.value === value;`